bower.json

```diff
@@ -10,6 +10,8 @@
     "lodash": "^4.10.0",
```

Comment:
I'd prefer to see versions hard-set, and not use `^` or `~`. Specify the exact version instead. This keeps the build stable. As new versions of libraries are released, feel free to update them, but then manually tag the exact version in the `package.json` file.